src/AndroidDebugLauncher/Launcher.cs

```diff
@@ -437,7 +437,7 @@ private LaunchOptions SetupForDebuggingWorker(CancellationToken token)
                 LaunchOptions launchOptions = null;
                 if (_targetEngine == TargetEngine.Native)
                 {
-                    launchOptions = new LocalLaunchOptions(_installPaths.GDBPath, string.Format(CultureInfo.InvariantCulture, ":{0}", gdbPortNumber), null);
+                    launchOptions = new LocalLaunchOptions(_installPaths.GDBPath, string.Format(CultureInfo.InvariantCulture, ":{0}", gdbPortNumber), String.Empty, null);
```

Comment:
>String.Empty [](start = 150, length = 12)

You might want to pass this along from `_launchOptions` instead

Reply:
Nevermind

---
In reply to: [187206988](https://github.com/Microsoft/MIEngine/pull/720#discussion_r187206988) [](ancestors = 187206988)